tests/lib/nm/dns_test.py

```diff
@@ -0,0 +1,130 @@
+#
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+import pytest
+
+from libnmstate import metadata
+from libnmstate.schema import DNS
+import libnmstate.nm.connection as nm_connection
+import libnmstate.nm.ipv4 as nm_ipv4
+import libnmstate.nm.ipv6 as nm_ipv6
+
+
+NM_USE_DEFAULT_DNS_PRIORITY = 0
```

Comment:
You should probably fetch it from `nm_ipv4` or `nm_ipv6`. Maybe the right thing is to move it to `dns.py` ?

Reply:
Cannot move it to `dns.py` for loop import concern in Python 2.7.

The `dns.py` import `ipv4.py` and `ipv6.py`. So `nm_ipv4` and `nm_ipv6` cannot use constants from `dns.py`.

Reply:
Wait a second. Checking.

Reply:
The python 2.7 does not support loop import.
So I have to detach the dns.py from `nm_ipv4` and `nm_ipv6`.  Then I can save the `DEFAULT_DNS_PRIORITY` in `dns.py`.